tests/ltp/ext2fs_fs_tests_passed.txt

```diff
@@ -134,10 +134,10 @@
 /ltp/testcases/kernel/syscalls/symlinkat/symlinkat01
 /ltp/testcases/kernel/syscalls/truncate/truncate02
 /ltp/testcases/kernel/syscalls/truncate/truncate02_64
+/ltp/testcases/kernel/syscalls/unlinkat/unlinkat01
```

Comment:
Can we add back the ltp tests disabled before the release? PR https://github.com/deislabs/mystikos/pull/621 is supposed to fix the regressions.

Reply:
Yes. I think this PR has gotten a little out of date. Will run the script again and update